code/mergeblock/foo1.log

```diff
@@ -0,0 +1,4 @@
+20
```

Comment:
I am 100% sure that log files won't be that simple. I would imagine it more like
```
1,2,2,2,2,2,2
3,3,1,,,1
```
or like (from example IMU or whatever)
```
acceleration_x=0.1,acceleration_y=0.2, acceleration_z=9.6
gyro_x=2.22,gyro_y=333
temp=44
```

Reply:
They are not going to be one value per line, thats for sure

Reply:
Our sensors' blocks will add to each value one-letter char ID - this will solve the problem